asc.jl

```diff
@@ -69,45 +70,48 @@ cᵖ = 3994.0   # [J K⁻¹] heat capacity
 
 parameters = (Ly = Ly,  
               Lz = Lz,    
+              y_salt_shutoff = - (Ly-50kilometers),# shutoff location for salt flux [km]
+              Qsalt = 2.5e-3,                      # salt input (into the domain) [gm^{-2}s^{-1}]
+              τ = 0.075/ρ,                         # surface kinematic wind stress [m² s⁻²]
               Qᵇ = 10 / (ρ * cᵖ) * α * g,          # buoyancy flux magnitude [m² s⁻³]    
               y_shutoff = 5/6 * Ly,                # shutoff location for buoyancy flux [m]
-              y_salt_shutoff = - Ly/4,             # shutoff location for buoyancy flux [m]
-              Qsalt = - 2.5e-3,                    # ... some units for salt flux
-              τ = 0.1/ρ,                           # surface kinematic wind stress [m² s⁻²]
-              μ = 1 / 30days,                      # bottom drag damping time-scale [s⁻¹]
+              ## TS14 use a linear drag coefficient of 1e-3 ms^{-1}, how does that compare to this? 
```

Comment:
linear drag below comes up as `- μ * Lz * u` so that implies a coefficient `μ * Lz = 1/30day * 3000m = 0.0011 m s⁻¹`. So I think this is very similar!